twoliter/embedded/kit2oci

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+for opt in "$@"; do
+    optarg="$(expr "${opt}" : '[^=]*=\(.*\)')"
+    case "${opt}" in
+        --content-dir=*) CONTENT_DIR="${optarg}" ;;
+        --architecture=*) ARCH="${optarg}" ;;
+        --output=*) OUTPUT="${optarg}" ;;
+    esac
+done
+
+# Create the content layer and digest
+# create_content_layer $TEMP_DIR
+create_content_layer() {
+    # First lets create the content archive
+    tar -cf $1/content-layer.tar -C $CONTENT_DIR .
+    gzip $1/content-layer.tar
+
+    DIGEST=$(cat $1/content-layer.tar.gz | sha256sum | awk '{ print $1 };')
+    mv $1/content-layer.tar.gz $1/blobs/sha256/$DIGEST
+    echo $DIGEST
+}
+
+# Create a layer for a configuration
+# create_json_layer $TEMP_DIR $JSON_BLOB
+create_json_layer() {
+    DIGEST=$(echo $2 | sha256sum | awk '{ print $1 };')
+    echo $2 > $1/blobs/sha256/$DIGEST
+    echo $DIGEST
+}
+
+# Create the config layer
+# create_config $TEMP_DIR $CONTENT_DIGEST $DATETIME
+create_config() {
+    DIFF_ID=$(gunzip < $1/blobs/sha256/$2 | sha256sum | awk '{ print $1 };' )
+    DATETIME=$3
+    METADATA=$(cat kit-metadata.json | base64)
```

Comment:
useless-use-of-cat nit, but also with `base64` you typically want to avoid the default line wrapping:
```suggestion
    METADATA=$(base64 <kit-metadata.json -w0)
```